tmt/steps/provision/__init__.py

```diff
@@ -462,6 +465,9 @@ def pull(self, source=None, destination=None, options=None):
             self.debug(f"Pull workdir from guest '{self.guest}'.")
         else:
             self.debug(f"Copy '{source}' from the guest to '{destination}'.")
+        # Make sure rsync is present, note: tests can remove it
+        self.debug('Ensure that rsync is installed on the guest.')
```

Comment:
```suggestion
        self.debug('Ensure that rsync is installed on the guest.', level=3)
```

Move to level 3? Probably not that much interesting debug message.